[ao][sparsity] README for data sparsifier lightning callbacks

macandro96 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #81813

The README contains instructions on using the lightning callbacks to sparsify the
model during and post training.

Test Plan: None

Commit:
[ao][sparsity] README for data sparsifier lightning callbacks

The README contains instructions on using the lightning callbacks to sparsify the
model during and post training.

Test Plan: None

[ghstack-poisoned]

z-a-f reviewed (approved):
Please, add signatures for the functions to let users know what each constructor/method/function is expecting and outputting.

Commit:
Update on "[ao][sparsity] README for data sparsifier lightning callbacks"


The README contains instructions on using the lightning callbacks to sparsify the
model during and post training.

Test Plan: None

[ghstack-poisoned]

Commit:
Update on "[ao][sparsity] README for data sparsifier lightning callbacks"


The README contains instructions on using the lightning callbacks to sparsify the
model during and post training.

Test Plan: None

[ghstack-poisoned]

macandro96 commented:
@pytorchbot merge -g
